README.md

```diff
@@ -437,14 +437,14 @@ describe the full use case. The fields which are not in ECS are in italic.
 
 Contributions of additional uses cases on top of ECS are welcome.
 
- * [APM](https://github.com/elastic/ecs/blob/master/use-cases/apm.md)
- * [Auditbeat](https://github.com/elastic/ecs/blob/master/use-cases/auditbeat.md)
- * [Beats](https://github.com/elastic/ecs/blob/master/use-cases/beats.md)
- * [Filebeat Apache](https://github.com/elastic/ecs/blob/master/use-cases/filebeat-apache-access.md)
- * [Kubernetes](https://github.com/elastic/ecs/blob/master/use-cases/kubernetes.md)
- * [Logging](https://github.com/elastic/ecs/blob/master/use-cases/logging.md)
- * [Metricbeat](https://github.com/elastic/ecs/blob/master/use-cases/metricbeat.md)
- * [TLS](https://github.com/elastic/ecs/blob/master/use-cases/tls.md)
+ * [APM](use-cases/apm.md)
```

Comment:
I'm wondering if these links still work if someone looks at the README directly from the landing page? Because then the link changes to `blob/master` so the relative link is not correct.

We should make sure these links keep working as I assume that are the ones most used.

Reply:
I tested all combinations, and it works in all cases:

- From a branch, clicking in root page rendering of readme towards use cases
- From a branch, clicking in /readme.md towards use cases
- From a branch, clicking from use cases back to field defs in the readme
- From root page of the repo (temporarily merged to master in my fork) to the readme and back

All of these cases